src/uu/sort/src/sort.rs

```diff
@@ -247,6 +247,7 @@ pub struct Output {
 }
 
 impl Output {
+    #[allow(clippy::suspicious_open_options)]
```

Comment:
please add a comment explaining the why

Reply:
humm here's a comment about why dont use `.truncate(true)` here

```
            // This is different from `File::create()` because we don't truncate the output yet.
            // This allows using the output file as an input file.
```